cc3d/core/PySteppables.py

```diff
@@ -667,6 +673,37 @@ def fetch_loaded_plugins(self) -> None:
                 for plugin_member_name in member_var_list:
                     setattr(self, plugin_member_name, None)
 
+    def fetch_loaded_steppables(self) -> None:
```

Comment:
I know this is a minor thing in the grand scheme of the reaction diff solver, but:

very cool